themes/ohcoe/static/js/scripts.js

```diff
@@ -151,4 +169,44 @@ $(function() {
     if (!localStorage.getItem('role') && !localStorage.getItem('speciality')) {
         $('#demographic-info').show();
     }
+
+    // Render on Cumulative Review page
+    if(document.getElementById('demographic-q-responses') &&
+        localStorage.getItem('role') &&
+        localStorage.getItem('speciality')) {
+        const DEMO_ANSWERS = {
+            role: {
+                prefer_not_answer: 'Prefer not to answer',
+                faculty: 'Faculty',
+                resident: 'Resident',
+                dentist: 'Dentist',
+                student: 'Other'
+            },
+            speciality: {
+                prefer_not_answer: 'Prefer not to answer',
+                public_health: 'Public Health',
+                general: 'General',
+                pediatric: 'Pediatric',
+            }
+        };
+
+        var role = DEMO_ANSWERS.role[localStorage.getItem('role')];
+        var speciality = DEMO_ANSWERS.speciality[
+            localStorage.getItem('speciality')];
+
+        // If 'other' is set, and there's no value in the DEMO_ANSWERS
+        // for the given question, then just use the value in storage.
+        if (!role){
```

Comment:
Some linting issues here.

Reply:
What was the linting issue? It didn't show up in my editor or eslint.

Reply:
oh weird, maybe we need to adjust the `.eslintrc` -- we usually follow this pattern:
`if (!role) {`

Reply:
ach, ok I'll fix it